docs/example-react-hooks-useContext.md

```diff
@@ -0,0 +1,92 @@
+This is a simple example to test a component using the useContext hook. 
+
+
+The context object
+
+```js
+import React, { createContext } from 'react'
+
+const Context = createContext()
+
+export default Context
+```
+
+the root parent component
+
+```jsx
+import React, { useState } from 'react'
+import Context from './context'
+
+
+const App = () => {
+  const [state, setState] = useState("Some Text")
+
+  const changeText = () => {
+    setState("Some Other Text")
+  }
+
+return (
+   <h1> Basic Hook useContext</h1>
+     <Context.Provider value={{changeTextProp: changeText,
+                               stateProp: state
+                                 }} >
+        <ChildComponent />
+     </Context.Provider>
+    </div>
+  );
+}
+
+export default App
+
+```
+
+
+the child component using the context
+
+```jsx
+import React, { useContext } from 'react'
+
+import Context from '../context'
+
+const ChildComponent = () => {
+  const context = useContext(Context)
+
+  return (
```

Comment:
Looks better! Thank you 🙌  could you please fix the indentation of the JSX elements?

```html
<div>
  <button ...>
    Change text
  </button>
  <p>...</p>
</div>
```